src/routes/robots.txt/+server.ts

```diff
@@ -0,0 +1,19 @@
+import type { RequestHandler } from "@sveltejs/kit";
+
+export const prerender = true;
```

Comment:
we dont prerender it, because it needs to be depenendent on the request, please set it to false 👍🏻